server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -632,7 +632,11 @@ TabletGoalState getGoalState(TabletMetadata tm) {
       }
 
       if (tm.hasCurrent() && serversToShutdown.contains(tm.getLocation().getServerInstance())) {
-        return TabletGoalState.SUSPENDED;
+        if (tgw.canSuspendTablets()) {
+          return TabletGoalState.SUSPENDED;
+        } else {
+          return TabletGoalState.UNASSIGNED;
+        }
```

Comment:
Style tip only (feel free to ignore):

```suggestion
        return tgw.canSuspendTablets() ? TabletGoalState.SUSPENDED : TabletGoalState.UNASSIGNED;
```